pkg/cmd/describe/origin/origin.go

```diff
@@ -28,24 +37,48 @@ var (
 func NewCmd(f *cmdutil.Factory) *cobra.Command {
 	opts := &contracts.DescribeOptions{}
 	cmd := &cobra.Command{
-		Use:           msg.OriginsDescribeUsage,
-		Short:         msg.OriginsDescribeShortDescription,
-		Long:          msg.OriginsDescribeLongDescription,
+		Use:           msg.Usage,
+		Short:         msg.ShortDescription,
+		Long:          msg.LongDescription,
 		SilenceUsage:  true,
 		SilenceErrors: true,
-		Example: heredoc.Doc(`
-      $ azion origins describe --application-id 1673635839 --origin-id 31223
-      $ azion origins describe --application-id 1673635839 --origin-id 31223--format json
-      $ azion origins describe --application-id 1673635839 --origin-id 31223--out "./tmp/test.json" --format json
-    `),
+		Example:       exemplo,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if !cmd.Flags().Changed("application-id") || !cmd.Flags().Changed("origin-id") {
-				return msg.ErrorMissingArguments
+			if !cmd.Flags().Changed("application-id") {
+				answers, err := utils.AskInput("What is the ID of the Edge Application?")
+				if err != nil {
+					logger.Debug("Error while parsing answer", zap.Error(err))
+					return utils.ErrorParseResponse
+				}
+
+				appID, err := strconv.Atoi(answers)
+				if err != nil {
+					logger.Debug("Error while parsing string to integer", zap.Error(err))
+					return utils.ErrorConvertingStringToInt
+				}
+
+				applicationID = int64(appID)
+			}
+
+			if !cmd.Flags().Changed("origin-id") {
```

Comment:
I don't remember why we did it like this. However you can GET an origin by its origin_key.
https://api.azion.com/#850d2a1d-98c2-4c29-91a0-8173ddaac8da